tests/openvino/utils_tests.py

```diff
@@ -141,6 +141,8 @@
     "xglm": "hf-internal-testing/tiny-random-XGLMForCausalLM",
     "xverse": "katuni4ka/tiny-random-xverse",
     "glm4": "katuni4ka/tiny-random-glm4",
+    "open-clip": "hf-internal-testing/tiny-open-clip-model",
+    "open-clip-ir": "zofinka/tiny-open-clip-model",
```

Comment:
for consistency with others (ir is very general name)
```suggestion
    "open-clip-ov": "zofinka/tiny-open-clip-model",
```

Reply:
renamed